rust/xaynet-server/src/state_machine/phases/mod.rs

```diff
@@ -139,22 +135,20 @@ where
 ///
 /// This contains the state-dependent `private` state and the state-independent `shared` state
 /// which is shared across state transitions.
-pub struct PhaseState<S, C, M>
+pub struct PhaseState<State, Store>
```

Comment:
personally for consistency i would say `<S, T>` and add a comment on what they represent, but I'm ok with this too if you prefer it.